archive/c/c/binary-search.c

```diff
@@ -114,7 +114,10 @@ char* trimWhitespace(char *str) {
 }
 
 // Function to handle errors
-void handleError(const char *message) {
+void handleError(const char *message, int usageMessage) {
     fprintf(stderr, "%s\n", message);
+    if (usageMessage) {
+        fprintf(stderr, "Usage: please provide a list of sorted integers (\"1, 4, 5, 11, 12\") and the integer to find (\"11\")\n");
+    }
```

Comment:
If usageMessage is 1, then this should not be displayed since this will still make the tests fail